azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/AzureStorageCheckpointLeaseManager.java

```diff
@@ -63,6 +65,8 @@
         // Convert all-whitespace prefix to empty string. Convert null prefix to empty string.
         // Then the rest of the code only has one case to worry about.
         this.storageBlobPrefix = (storageBlobPrefix != null) ? storageBlobPrefix.trim() : "";
+        
+        this.mgrThreadpool = Executors.newCachedThreadPool(new NamedThreadFactory("asclm-"));
```

Comment:
lets expose `this.mgrThreadPool` to be set by users.. as something like - `LeaseTaskExecutor` or a better word!